index.js

```diff
@@ -360,8 +367,18 @@ function initInput() {
   });
 
   $('#run').addEventListener('click', run);
+  $('#stop').addEventListener('click', stop);
   $('#clear').addEventListener('click', clear);
 
+  function setRun(canRun) {
+    if (canRun) {
```

Comment:
A better approach is to set/clear a style on an ancestor (e.g. `document.body.classList.add('running')` / `document.body.classList.remove('running')`) and then use CSS to control the visibility:

```css
#run, .running #stop { display: block; }
#stop, .running #run { display: none; }
```